packages/sui-mono/src/tags.js

```diff
@@ -0,0 +1,49 @@
+const {exec} = require('child_process')
+
+function checkSUITag(tag, pkg) {
+  if (pkg && !tag.startsWith(`${pkg}-`)) {
```

Comment:
Here we filter sui package tags

Reply:
We should add the comments in the code, what do you think?